scippy/tests/test_units_neutron.py

```diff
@@ -0,0 +1,23 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+# Copyright (c) 2019 Scipp contributors (https://github.com/scipp)
+# @file
+# @author Simon Heybrock
+import scippy as sp
+from scippy import Dim
+
+
+def test_dims():
```

Comment:
not sure this should be part of the tests on `units`? Move to `test_dimensions.py`?

Reply:
`test_dimensions.py` tests `class Dimensions`, this is the labels, which are part of `scipp::units`.